src/statsig/index.ts

```diff
@@ -142,7 +142,7 @@ export function setupOverridesFromLaunchArgs() {
     const { statsigGateOverrides } = LaunchArguments.value<ExpectedLaunchArgs>()
     if (statsigGateOverrides) {
       Logger.debug(TAG, 'Setting up gate overrides', statsigGateOverrides)
-      statsigGateOverrides.split(',').forEach((gateOverride) => {
+      statsigGateOverrides.split(',').forEach((gateOverride: string) => {
```

Comment:
is this change required? `split`'s type should guarantee `string[]` as the return type right?

Reply:
I'm not sure why this was needed; my TS compiler was yelling at me without it.